src/parser/mod.rs

```diff
@@ -2390,14 +2394,14 @@ impl<'a> Parser<'a> {
         let old_value = self.options.trailing_commas;
         self.options.trailing_commas |= dialect_of!(self is BigQueryDialect);
 
-        let ret = self.parse_comma_separated(|p| p.parse_select_item());
+        let ret = self.parse_comma_separated1(|p| p.parse_select_item());
         self.options.trailing_commas = old_value;
 
         ret
     }
 
     /// Parse a comma-separated list of 1+ items accepted by `F`
-    pub fn parse_comma_separated<T, F>(&mut self, mut f: F) -> Result<Vec<T>, ParserError>
+    pub fn parse_comma_separated1<T, F>(&mut self, mut f: F) -> Result<Vec<T>, ParserError>
```

Comment:
given this is a public function, can you please keep the name the same (I see why it would make sense to name it consistently with `parse_comma_separated0` but in this case I think it would be better not to make such a large scale function name change)

Reply:
Ok, changed back.